src/main/java/cohen/mvc/FreebiePostsFrame.java

```diff
@@ -188,19 +189,21 @@ public void valueChanged(ListSelectionEvent e)
         mainPanel.add(postTitles, BorderLayout.WEST);
         mainPanel.add(individualPost, BorderLayout.CENTER);
 
-        controller.refreshPosts(userLat.getText(), userLon.getText(), getUTCDateParam());
+        controller.refreshPosts(userLat.getText(), userLon.getText(), getUtcDateParam());
 
         setContentPane(mainPanel);
     }
 
-    private String getUTCDateParam()
+    private String getUtcDateParam()
     {
         return userDate.getText() + "T00%3A00%3A00";
```

Comment:
The `%3A` is the ":" character, just escaped. You should be using ":" instead.
https://agileappscloud.info/aawiki/UTC_Format

Reply:
Right, thank you. Thought it needed to be escaped but that would make more sense, and is probably why the service was having issues